sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -244,6 +245,24 @@ rename-mapping:
   MongoIndexOptions: MongoDBIndexConfig
   BackupStorageRedundancy: CosmosDBBackupStorageRedundancy
   PrimaryAggregationType: CosmosDBMetricPrimaryAggregationType
+  RestorableSqlResourcesGetResult: RestorableSqlResourceData
+  RestorableMongodbResourcesGetResult: RestorableMongoDBResourceData
+  ServiceResourceProperties: CosmosDBServiceProperties
+  ServiceResourceCreateUpdateParameters: CosmosDBServiceCreateUpdateParameters
+  ServiceResource: CosmosDBService
+  ServiceResourceListResult: CosmosDBServiceListResult
+  DataTransferServiceResourceProperties: DataTransferServiceProperties
+  SqlDedicatedGatewayServiceResourceProperties: SqlDedicatedGatewayServiceProperties
+  GraphAPIComputeServiceResourceProperties: GraphAPIComputeServiceProperties
```

Comment:
```suggestion
  GraphAPIComputeServiceResourceProperties: GraphApiComputeServiceProperties
```
Update the casing of `Api`. If you work on Windows, you may need to delete the following 4 files: `GraphAPIComputeRegionalService.cs`, `GraphAPIComputeRegionalService.Serialization.cs`, `GraphAPIComputeServiceProperties.Serialization.cs`, `GraphAPIComputeServiceProperties.cs`. Then commit the change and regenrate.  Otherwise the casing in file names will not be updated since Windows file names are case-insensitive.